dashboard/app/views/lti/v1/link_existing_account.html.haml

```diff
@@ -0,0 +1,30 @@
+!!!
```

Comment:
Please place the view files under the resource/controller namespace:
`dashboard/app/views/lti/v1/users/link_existing_account.html.haml`